odoo/addons/base/views/ir_actions_views.xml

```diff
@@ -472,6 +472,55 @@ env['res.partner'].create({'name': partner_name})
         </record>
         <menuitem action="action_server_action" id="menu_server_action" parent="base.next_id_6"/>
 
+        <!-- ir.embedded.actions -->
+
+        <record id="embedded_action_form" model="ir.ui.view">
+            <field name="name">ir.embedded.actions.form</field>
+            <field name="model">ir.embedded.actions</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet string="Embedded Actions">
+                        <group>
+                            <group>
+                                <field name="name" string="Name"/>
+                                <field name="parent_action_id" can_create="True" can_write="True"/>
```

Comment:
remove can_create, can_write and apply "strings" directly in the model definition